_schemaV2.graphql

```diff
@@ -12447,6 +12450,18 @@ type SearchCriteriaEdge {
   node: SearchCriteria
 }
 
+# Human-friendly representation of a single SearchCriteria filter
+type SearchCriteriaLabel {
+  # The GraphQL field name of the filter facet
+  field: String
+
+  # The human-friendly name of the filter facet
+  name: String
+
+  # The human-friendly value of the filter facet
+  value: String
+}
+
```

Comment:
This is a new type defined here in MP, and is used to define the new `labels` field which is inserted into the `SearchCriteria` type:

<img height="500" alt="label" src="https://user-images.githubusercontent.com/140521/153664789-003e1458-95d0-428c-a520-28170e81ccf3.png">